command/operator_debug.go

```diff
@@ -69,9 +71,15 @@ Debug Options:
   -log-level=<level>
     The log level to monitor. Defaults to DEBUG.
 
+  -max-nodes=<count>
```

Comment:
I'm trying to see if there is any precedent for having unlimited be -1, so that you can actually specify 0

Reply:
If you don't specify `-node-id` then `-max-nodes` doesn't apply.  In testing though I discovered that the max node count notice is incorrectly displayed in that case.